lib/oxidized/node.rb

```diff
@@ -231,11 +233,13 @@ def resolve_key key, opt, global=nil
     end
 
     def is_git? opt
-      (opt[:output] || Oxidized.config.output.default) == 'git'
+      outputs = (opt[:output] || Oxidized.config.output.default)
+      outputs.split(/\s*,\s*/).map.include?('git')
```

Comment:
this `#split` is repeated several times. could you extract it to a private method? something like:

``` ruby
def output_list
  @output_list ||= outputs.split(/\s*,\s*/)
end
```